spark/src/main/scala/com/acervera/osm4scala/spark/OsmPbfFormat.scala

```diff
@@ -122,7 +122,12 @@ class OsmPbfFormat extends FileFormat with DataSourceRegister {
 
   override def inferSchema(sparkSession: SparkSession,
                            options: Map[String, String],
-                           files: Seq[FileStatus]): Option[StructType] = Some(OsmSqlEntity.schema)
+                           files: Seq[FileStatus]): Option[StructType] =
+    if (options.getOrElse("wayWithGeometry","false").toLowerCase().equals("true")) {
```

Comment:
I would keep only one schema.
After this change , I think that I will increase the major version of the library, because back compatibility (and I will take this chance to drop Scala 2.11 comp).
I don't think that adding two nullable columns will affect too much.

Reply:
Just wanted to keep it backward compatible. I can keep only one if you think it's better